locobot/robot/remote_locobot.py

```diff
@@ -158,14 +156,11 @@ def go_to_absolute(self, xyt_position, use_map=False, close_loop=True, smooth=Fa
         :type close_loop: bool
         :type smooth: bool
         """
-        if self._done:
-            self._done = False
-            self._robot.base.go_to_absolute(
-                xyt_position, use_map=use_map, close_loop=close_loop, smooth=smooth
-            )
-            self._done = True
+        if self._move_done:
+            self._move_done = False
+            self._slam.set_absolute_goal_in_pr_frame(xyt_position)
```

Comment:
does pr stand for pyrobot? in the medium to long term, we'll want to use this for non-pyrobot platforms as well. Rename this to something else like `set_absolute_goal_in_robot_frame` perhaps.

Reply:
updated `pr_frame` to `robot_frame`